frontend-ssr/redux/exercise-submissions/exercise-submissions.reducer.ts

```diff
@@ -0,0 +1,78 @@
+import { SubmissionState } from './types';
+import { EXERCISE_SUBMISSIONS } from './exercise-submissions.actions';
+import MOCK_SUBMISSIONS from '~/pages/api/exercise-submissions/mock';
+
+const initialState: SubmissionState = {
+  submissions: undefined,
+  search: '',
+  page: 1,
```

Comment:
Let's use 0-based index for page number. Otherwise it may be confusing, since almost all things in JS start from index 0.